### DISCUSSION TITLE
23.10 "American Bobtail"

### DISCUSSION THREAD

**Discussion by Nekosis** [Meowix-Linux/Meowix-ISO#9] [Announcements]
<p align="center">
<img src="https://github.com/Meowix-Linux/Meowix-ISO/blob/main/assets/meowix-text.svg?raw=true" width="50%" height="50%">
</p>

<h1 align="center">Meowix Linux 23.10 "American Bobtail"</h1>

<p align="center">Welcome to version 23.10 of Meowix Linux, proudly presented by the Meowix maintainers.</p>

<p align="center">Please note that Meowix is currently in its early development stages, which may result in occasional instability. We value your participation and encourage you to <a href="https://github.com/Meowix-Linux/Meowix-ISO/issues">report any bugs or issues</a> you encounter.</p>

## Notable Changes

In this release, we are excited to introduce:

- **Neovim by Default**: We've added Neovim to this month's ISO. It's now set as the default text editor, replacing Micro, with a simple yet effective configuration and convenient aliases included out of the box, enhancing your development experience. *Note: Micro is still installed and configured out of the box, but is not set as the default text editor. If you'd like to change this, edit `~/.bashrc` after installation.*

- **User Experience Improvements**: The team has worked on improving keybindings for switching windows and workspaces, making navigation smoother for users.

- **Refined Logo**: We've merged PR #5, which entailed a cleaner icon for Meowix to represent its brand identity. Thank you for the design, @andreasgrafen!

## New Contributors
- @andreasgrafen made their first contribution in https://github.com/Meowix-Linux/Meowix-ISO/pull/5

##

Your support and engagement are vital to the ongoing development of Meowix Linux. Thank you for being part of our community.

We look forward to your feedback and contributions as we continue refining and expanding Meowix Linux.

<hr /><em>This discussion was created from the release <a href='https://github.com/Meowix-Linux/Meowix-ISO/releases/tag/23.10'>23.10 "American Bobtail"</a>.</em>

**Comment by unseen-ninja** [comment]
![Peepo Stonks](https://cdn.betterttv.net/emote/60fdac4d2d1eba5400d17c8d/2x.webp)
